plain_http_producer_test.go

```diff
@@ -41,6 +41,7 @@ func TestSendMessage(t *testing.T) {
 				"X-Origin-System-Id": "methode-web-pub",
 				"X-Request-Id":       "t9happe59y",
 				"Authorization":      "authorizationkey",
+				"Message-Timestamp":  "2015-07-06T07:03:09.362Z",
```

Comment:
Following the existing pattern here, I think Message-Timestamp should be included in the expected headers for all the appropriate test cases.